### DISCUSSION TITLE
Customize /Reports section of widget.

### DISCUSSION THREAD

**Discussion by senthilkumardhanapal2** [Azure/api-management-developer-portal#2599] [Feature request]
Hello,

as part of developer portal, we see the /reports  below and would like to customize the reports widget ( remove Products and Subscription section from the widget.). is there a we could do that from managed version of developer portal ?

https://integtzinterop.developer.healthtranzformdev.com/reports
![image](https://user-images.githubusercontent.com/81538941/183492752-bbeca82f-f1f5-4259-befb-fae1b2043abb.png)

**Comment by ghost** [comment]
@senthilkumardhanapal2, thank you for opening this issue. We will triage it within the next few business days.

**Comment by azaslonov** [comment]
@senthilkumardhanapal2, not at the moment, but I think your request makes sense.

**Comment by ghost** [comment]
@senthilkumardhanapal2, thank you for requesting this feature.

**Comment by Mdshakeelse** [comment]
Wanted to know how the search functionality is implemented here
